app/views/helpers/index/normal/entry_header.phtml

```diff
@@ -52,26 +53,72 @@
 	?></li><?php
 	endif; ?>
 	<li class="item titleAuthorSummaryDate">
-	<span class="item-element title<?= (($topline_thumbnail !== 'none') || $topline_summary) ? ' multiline' : '' ?>" dir="auto"><a target="_blank" rel="noreferrer" href="<?= $this->entry->link() ?>"><?= $this->entry->title() ?><?php
-		if ($topline_display_authors):
-			?><span class="author"><?php
-			$authors = $this->entry->authors();
-			if (is_array($authors)) {
-				$first = true;
-				foreach ($authors as $author) {
-					echo $first ? $author : ', ' . $author;
-					$first = false;
+		<span class="item-element title<?= (($topline_thumbnail !== 'none') || $topline_summary) ? ' multiline' : '' ?>" dir="auto"><a target="_blank" rel="noreferrer" href="<?= $this->entry->link() ?>"><?= $this->entry->title() ?><?php
+			if ($topline_display_authors):
+				?><span class="author"><?php
+				$authors = $this->entry->authors();
+				if (is_array($authors)) {
+					$first = true;
+					foreach ($authors as $author) {
+						echo $first ? $author : ', ' . $author;
+						$first = false;
+					}
 				}
-			}
-			?></span><?php
+				?></span><?php
+			endif;
+			?>
+		</a></span>
+		<?php
+		if ($topline_summary):
+			?><div class="summary"><?= trim(mb_substr(strip_tags($this->entry->content(false)), 0, 500, 'UTF-8')) ?></div><?php
 		endif;
-	?></a></span>
-	<?php
-	if ($topline_summary):
-		?><div class="summary"><?= trim(mb_substr(strip_tags($this->entry->content(false)), 0, 500, 'UTF-8')) ?></div><?php
-	endif; ?>
+	?>
 	<?php if ($topline_date) { ?><span class="item-element date"><time datetime="<?= $this->entry->machineReadableDate() ?>"><?= $this->entry->date() ?></time>&nbsp;</span><?php } ?>
 	</li>
+	<?php if ($topline_sharing) { ?>
+		<li class="item share">
+			<div class="item-element dropdown">
+				<div id="dropdown-share2-<?= $this->entry->id() ?>" class="dropdown-target"></div>
+				<a class="dropdown-toggle" href="#dropdown-share2-<?= $this->entry->id() ?>" title="<?= _t('index.share') ?>">
+					<?= _i('share') ?>
+				</a>
+
+				<ul class="dropdown-menu">
+					<li class="dropdown-header"><?= _t('index.share') ?> <a href="<?= _url('configure', 'integration') ?>"><?= _i('configure') ?></a></li><?php
+						$id = $this->entry->id();
+						$link = $this->entry->link();
+						$title = $this->entry->title() . ' · ' . ($this->feed === null ? '' : $this->feed->name());
+						foreach (FreshRSS_Context::userConf()->sharing as $share_options) {
```

Comment:
Implementation consideration: this adds a lot of data in the output (consume resources server side, at network level, and client side) for a non-essential shortcut feature. To save resources and improve performances, I think this should be left to a dynamically-generated menu like we have for the configuration drop-down menu of the feeds in the aside bar.

Example:
https://github.com/FreshRSS/FreshRSS/blob/329fd4bcf6504c74e3906e51c6fc2124bc09cc02/app/layout/aside_feed.phtml#L153-L189

Or (probably even better) use an HTML template element https://developer.mozilla.org/docs/Web/HTML/Element/template like:
https://github.com/FreshRSS/FreshRSS/blob/329fd4bcf6504c74e3906e51c6fc2124bc09cc02/app/views/configure/integration.phtml#L15-L31